### ISSUE TITLE
Where's the file to change the copyright year in qat_prov.txt, qat_prov_err.c and qat_prov_err.h?

### ISSUE THREAD

**Issue by Syllinia** [intel/QAT_Engine#322]
Each time after I compile QAT engine, these 3 files appear in the changes and it's a bit hard to find the file modifying the copyright year. Could you help to pick it out? Thanks.

Seems util/mkerr.pl is related with the copyright year modification where $YEAR is used.
perl -I/xxx/openssl-openssl-3.1.4-quic1 /yyyy/openssl-openssl-3.1.4-quic1/util/mkerr.pl -conf qat_prov.ec \
         -rebuild -reindex `ls *.c | sed "s/qat_bssl_err.c//"`

$git status
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   qat_prov.txt
        modified:   qat_prov_err.c
        modified:   qat_prov_err.h

diff --git a/FortiWEB/migbase/hwssl/intel-ssl/QAT_Engine/qat_prov.txt b/FortiWEB/migbase/hwssl/intel-ssl/QAT_Engine/qat_prov.txt
index 1d0165c1003..576135b953a 100644
--- a/FortiWEB/migbase/hwssl/intel-ssl/QAT_Engine/qat_prov.txt
+++ b/FortiWEB/migbase/hwssl/intel-ssl/QAT_Engine/qat_prov.txt
@@ -1,4 +1,4 @@
-# Copyright 1999-2023 The OpenSSL Project Authors. All Rights Reserved.
+# Copyright 1999-2024 The OpenSSL Project Authors. All Rights Reserved.
 #
 # Licensed under the Apache License 2.0 (the "License").  You may not use
 # this file except in compliance with the License.  You can obtain a copy
diff --git a/FortiWEB/migbase/hwssl/intel-ssl/QAT_Engine/qat_prov_err.c b/FortiWEB/migbase/hwssl/intel-ssl/QAT_Engine/qat_prov_err.c
index 9d75e258657..adc0c119539 100644
--- a/FortiWEB/migbase/hwssl/intel-ssl/QAT_Engine/qat_prov_err.c
+++ b/FortiWEB/migbase/hwssl/intel-ssl/QAT_Engine/qat_prov_err.c
@@ -1,6 +1,6 @@
 /*
  * Generated by util/mkerr.pl DO NOT EDIT
- * Copyright 1995-2023 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 1995-2024 The OpenSSL Project Authors. All Rights Reserved.
  *
  * Licensed under the Apache License 2.0 (the "License").  You may not use
  * this file except in compliance with the License.  You can obtain a copy
diff --git a/FortiWEB/migbase/hwssl/intel-ssl/QAT_Engine/qat_prov_err.h b/FortiWEB/migbase/hwssl/intel-ssl/QAT_Engine/qat_prov_err.h
index 53bdf234fa3..2fa29a17b44 100644
--- a/FortiWEB/migbase/hwssl/intel-ssl/QAT_Engine/qat_prov_err.h
+++ b/FortiWEB/migbase/hwssl/intel-ssl/QAT_Engine/qat_prov_err.h
@@ -1,6 +1,6 @@
 /*
  * Generated by util/mkerr.pl DO NOT EDIT
- * Copyright 1995-2023 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 1995-2024 The OpenSSL Project Authors. All Rights Reserved.
  *
  * Licensed under the Apache License 2.0 (the "License").  You may not use
  * this file except in compliance with the License.  You can obtain a copy

**Comment by venkatesh6911** [comment]
Hi @Syllinia , can you provide the QAT Engine version along with the configure command used to build it ?

**Comment by Syllinia** [comment]
QAT Engine version is v1.5.0.

        ./configure --host=x86_64-linux \
        --with-qat_hw_dir=$(TOPDIR)/kernel/modules/qat_driver \
        --with-qat_hw_install_dir=$(istldir) \
        --with-openssl_dir=$(curdir)/$(openssl_pkg) \
        --with-openssl_install_dir=$(curdir)/$(openssl_pkg) \
        --enable-qat_hw_multi_thread \
        --enable-qat_hw_contig_mem \
        --disable-qat_hw_lenstra_protection ; \

**Comment by sharanakumarpatil** [comment]
Right, copyright $YEAR is updated in the openssl/util/mkerr.pl file by the current system's year (2024).

It's triggered by "make err-files".
One way to avoid this trigger is to skip adding "--with-openssl_dir=$(curdir)/$(openssl_pkg)" with  ./configure.

**Comment by venkatesh6911** [comment]
Closing the ticket due to no activity.
